File: backend-portifolio/Models/Title.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend_portifolio.Models
{
    public class Title
    {
        public long Id {get; set;}
        public Guid Guid {get; set;}
        public int Sequence {get; set;}
        public string Content {get; set;}
    }
}

File: portfolio-v3/src/domain/models/Title.ts
import { Section } from "./Section";

export class Title{
    public Id: number;
    public Guid: string;
    public Sequence: number;
    public Content: string;
    public SectionId: number;
    public Section: Section;
    constructor(){
        this.Id = 0;
        this.Guid = "";
        this.Sequence = 0;
        this.Content = "";
        this.SectionId = 0;
        this.Section = new Section()
    }
}